package.json

```
@@ -10,7 +10,10 @@
   "scripts": {
     "test": "jest --verbose",
     "test:cover": "jest --coverage",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "build": "npm run rollup",
+    "rollup": "npm run rollup:esm",
+    "rollup:esm": "rollup -c rollup.config.esm.js"
```

I think we can simplify this.

```suggestion
    "build": "rollup -c rollup.config.esm.js"
```